### PR TITLE
Fixing infinite loop issue which occurred due to wrong fetch from array

### PR DESCRIPTION
**Pull request by dr86ansixavier**

### CODE REVIEW

File: src/main/index.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export namespace ObjectMapper {
             converstionFunctionsArray.push(struct);
         });
 
-        let conversionFunctionStructure: ConversionFunctionStructure = converstionFunctionsArray[0];
+        let conversionFunctionStructure: ConversionFunctionStructure = (converstionFunctionsArray.length > 0)? converstionFunctionsArray.pop() : undefined;
```
Comment (dr86ansixavier) [inline src/main/index.ts:44]: This is the actual fix. The other changes are just updates.

Comment (dr86ansixavier) [reply src/main/index.ts:44]: @shakilsiraj Can you review this PR and merge it to your base branch if it make sense to you?

Comment (dr86ansixavier) [reply src/main/index.ts:44]: Refer this ticket for more details https://github.com/shakilsiraj/json-object-mapper/issues/80. If you need any additional details, let me know.

**Comment by xavierdevairakkam** [comment]
@shakilsiraj Any updates? Can we merge this bug fixes and deploy this?

File: src/main/index.ts (hunk 1 above)
Comment (vapkse) [reply src/main/index.ts:44]: Hello. Possible to merge this fix, and make a release please?

**Review by xavierdevairakkam — APPROVED** [review]

File: src/main/index.ts (hunk 1 above)
Comment (dr86ansixavier) [reply src/main/index.ts:44]: yeah, we can merge this fix. It has been tested well and we are using this branch in few of our projects. Unfortunately, I don't have access to merge this PR. Can you please help in merge the branch and take a new release?

**Review by vapkse — APPROVED** [review]

File: src/main/index.ts (hunk 1 above)
Comment (dr86ansixavier) [reply src/main/index.ts:44]: @vapkse Can you merge this branch to master if you have access to merge?

Comment (vapkse) [reply src/main/index.ts:44]: I don't have access to merge.

Comment (xavierdevairakkam) [reply src/main/index.ts:44]: Is it @shakilsiraj the one can merge? Do you suggest anybody else if you know?

**Comment by vapkse** [comment]
Release 1.7.2 is available

**Comment by xavierdevairakkam** [comment]
Awesome. Thank you guys.

**Pull request merged**